documentation/seed-node.md

```diff
@@ -0,0 +1,255 @@
+# Seed node commands
+
+Vulcanizedb can act as an index for Ethereum data stored on IPFS through the use of the `syncAndPublish` and
+`syncPublishScreenAndServe` commands. 
+
+## Setup
+
+These commands work in conjunction with a [state-diffing full Geth node](https://github.com/vulcanize/go-ethereum/tree/rpc_statediffing)
+and IPFS.
+
+### IPFS
+To start, download and install [IPFS](https://github.com/vulcanize/go-ipfs)
+
+`go get github.com/ipfs/go-ipfs`
+
+`cd $GOPATH/src/github.com/ipfs/go-ipfs`
+
+`make install`
+
+If we want to use Postgres as our backing datastore, we need to use the vulcanize fork of go-ipfs. This fork supports
+the Postgres datastore plugin and has been adjusted to use `dep` instead of Go modules since Go modules cannot work with
+un-versioned forks and we need to use an un-versioned fork of go-ipfs-config.
+
+Start by adding the fork and switching over to it:
+
+`git remote add vulcanize https://github.com/vulcanize/go-ipfs.git`
+
+`git fetch vulcanize`
+
+`git checkout -b postgres_update vulcanize/postgres_update`
+
+Now install this fork of ipfs, first be sure to remove any previous installation. It is important to use the below command
+instead of using `make install`, as `make install` will default to using Go modules and this will wreck our dependencies
+since Go modules cannot work with un-versioned forks.
+
+`go install ./cmd/ipfs`
+
+Check that is installed properly by running
+
+`ipfs`
+
+You should see the CLI info/help output.
+
+And now we initialize with the `postgresds` profile.
```

Comment:
What is the `postgresds` profile? Is it telling ipfs to run with postgres as its backing datastore?

Reply:
Yeah exactly!

Reply:
Why do we have a particular interest in what datastore ipfs uses? Might have misunderstood things, but wouldn't the chain data be stored on IPFS and cids distributed to clients asking for certain data. Then, can't we store cids and relevant metadata in whatever database locally, without having to fiddle with how IPFS stores things?

Reply:
By having the IPFS data in the very same Postgres DB that VulcanizeDB lives in, it would enable us to write PL-WASM transformers which operate entirely within Postgres with no need to reach out to an external database.

Reply:
It also saves us from storing the data twice.

Reply:
Cool, cheers